lib/config.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+var Promise = require("bluebird");
+var Confidence = require("confidence");
+var Joi = require("joi");
+
+var configSettings;
+
+function Config() {
+	configSettings = {
```

Comment:
Ed's intent is to make Config instances immutable. I think it makes sense for each instance to not necessarily share state. That said, moving `configSettings` declaration to inside the `Config` function would be idea.